packages/office-addin-manifest/src/manifestInfo.ts

```diff
@@ -79,9 +80,17 @@ export async function modifyManifestFile(manifestPath: string, guid?: string, di
     if (guid === undefined && displayName === undefined) {
       throw new Error("You need to specify something to change in the manifest.");
     } else {
-      manifestData = await modifyManifestXml(manifestPath, guid, displayName);
-      await writeManifestData(manifestPath, manifestData);
-      return await readManifestFile(manifestPath);
+      try {
+        manifestData = await modifyManifestXml(manifestPath, guid, displayName);
+        await writeManifestData(manifestPath, manifestData);
+        let output = await readManifestFile(manifestPath);
+        sendUsageDataSuccessEvent("modifyManifestFile");
+        return output;
+        
+      } catch(err) {
+        sendUsageDataException("modifyManifestFile", err);
+        throw err;
```

Comment:
I think this block of code may be the source for the CI loop failure we are seeing on Mac, although I am not sure why it's failing on Mac in particular.  The problem looks to be that when we throw an error we include the full filepath in the error message, and the test is not expecting the full file path.  We should update the test to reflect this code change

Reply:
The error in our pipeline is because the tests expect the full filepath but instead just get `manifest.xml`. I tried running the manifest tests on a separate Mac and got all passing, for some reason

Reply:
<img width="1920" alt="Screen Shot 2020-02-06 at 4 21 36 PM" src="https://user-images.githubusercontent.com/3607629/73990088-f35ae400-48fc-11ea-84bc-bc169a76107a.png">